pkg/rtc/mediatracksubscriptions.go

```diff
@@ -110,7 +110,7 @@ func (t *MediaTrackSubscriptions) AddSubscriber(sub types.LocalParticipant, wr *
 	downTrack, err := sfu.NewDownTrack(
 		codecs,
 		wr,
-		t.params.BufferFactory,
+		sub.GetBufferFactory(),
```

Comment:
Can we remove this from params?

Reply:
removed. [e050cc9](https://github.com/livekit/livekit/pull/1195/commits/e050cc93c82dc87c4cfe5f6033cdd60c8999f8db)